drivers/dma/dma_stm32.c

```diff
@@ -203,6 +249,49 @@ static int dma_stm32_configure(struct device *dev, u32_t id,
 	const struct dma_stm32_config *dev_config =
 					dev->config->config_info;
 	DMA_TypeDef *dma = (DMA_TypeDef *)dev_config->base;
+
+#ifdef CONFIG_DMAMUX_STM32
+	/*
+	 * Also configures the corresponding dmamux channel
+	 * This dmamux channel is automatically enabled
+	 * by this dma configuration
+	 */
+	struct device *dev_mux =
+		   device_get_binding((const char *)"DMAMUX_1");
+
+	if (dev_mux == NULL) {
+		LOG_ERR("cannot get dmamux device.");
+		return -EINVAL;
+	}
+
+	const struct dmamux_stm32_config *devmux_config =
+					dev_mux->config->config_info;
+
+	/*
+	 * Retrieve the dmamux channel from the dma instance
+	 * and channel (stream)
+	 */
+	u32_t mux_id = get_dmamux_channel_from_dma(id);
```

Comment:
Now it is known, can we store `mux_id` in a dedicated `dma_stm32_data` field ?

Reply:
done,
adding a field `mux_id` to the `dma_stm32_stream` structure